Make LambertW.jl Pkg(3) ready. No dependency on Compat. Version set to 0.4.1 (should match tag in METADATA)

goedman opened the pull request:
Items in this PR:

1. Updated .travis.yml.
2. Added Manifest.toml to .gitignore.
3. Updated runtests.jl to show test summary and added extra line
4. Added Project.toml file

jlapeyre commented on test/runtests.jl:
```diff
@@ -1,4 +1,6 @@
-include("lambertw_test.jl")
+@testset "LambertW.jl" begin
+  include("lambertw_test.jl")
+end
```
I think there is a final newline missing here.

jlapeyre commented on Project.toml:
```diff
@@ -0,0 +1,7 @@
+name = "LambertW"
```
Why is `Compat` added as a dependency ?  I don't think it is used anywhere.

jlapeyre replied:
I was already using a Project.toml file locally, but with different info. I can put it in later, or you can include it
```
desc = "Lambert W function and associated omega constant"
license = "MIT"
name = "LambertW"
```

goedman replied:
I'll put it in later today, remove Compat and also check some more with the registered version of LambertW.

jlapeyre replied:
Great! I haven't taken the time yet to test how the registered v0.4 interacts with my system.

goedman commented:
Hi John, I think I have addressed all issues/suggestions. Let me know if this would work for you. Next step will be the PR for Symata.jl for Pkg(3) and then a 2nd PR for Symata using the package LambertW.jl. Should not be too hard.

jlapeyre commented:
Thanks! I squashed the commit: https://github.com/jlapeyre/LambertW.jl/commit/819fa55f5aef2f396993645b388eefb7944f9ae3
